notebooks/BTA_array_producer.ipynb

```diff
@@ -0,0 +1,2322 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.            "pileup_tightID": ak.values_astype(
</pre><p>Be aware that pileup jet IDs for Run 3 are WIP and likely to be needed only for forward jets, see [JetMET TWiki](https://twiki.cern.ch/twiki/bin/viewauth/CMS/JetMET)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cms-btv-pog/BTVNanoCommissioning/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
With current [PFNano](https://github.com/cms-jet/PFNano/tree/13_0_7_from124MiniAOD), we don't have this. Thanks for point this out. We should keep this in mind